### ISSUE TITLE
GraphEdit node connections aren't antialiased in running projects (only in the editor)

### ISSUE THREAD

**Issue by Nolkaloid** [godotengine/godot#17471]
**Godot version:**
3.0.2 Stable

**OS/device including version:**
Ubuntu 16.04

**Issue description:**
The GraphEdit connections are not smoothed out:
![image](https://user-images.githubusercontent.com/30960698/37302745-b60d893a-262c-11e8-9fd4-be3c57d1b8df.png)

While they are in the VS editor:
![image](https://user-images.githubusercontent.com/30960698/37302812-e16df754-262c-11e8-9e54-c0a7c2c531f3.png)

**Steps to reproduce:**
Set up a Graph Edit, add some nodes and connect them.

**Comment by Nobelix** [comment]
Ehhh nope, still not fixed 
![capture d ecran de 2019-01-06 13-14-46](https://user-images.githubusercontent.com/30960698/50735831-22ac9800-11b5-11e9-94b8-e21eae00d4c0.png)

**Comment by akien-mga** [comment]
This actually seems to be a driver-specific issue, for me the curves are antialiased when I use my AMD dGPU (Radeon RX Vega M), but not when I use the Intel HD Graphics 630 on the same laptop.
```
System:    Host: cauldron Kernel: 5.3.7-desktop-4.mga7 x86_64 bits: 64 Desktop: KDE Plasma 5.15.4 Distro: Mageia 7 mga7 
Graphics:  Device-1: Intel HD Graphics 630 driver: i915 v: kernel 
           Device-2: Advanced Micro Devices [AMD/ATI] Polaris 22 XL [Radeon RX Vega M GL] driver: amdgpu v: kernel 
           Display: x11 server: Mageia X.org 1.20.5 driver: amdgpu,intel FAILED: ati unloaded: fbdev,modesetting,vesa tty: N/A 
           OpenGL: renderer: Mesa DRI Intel HD Graphics 630 (Kaby Lake GT2) v: 4.5 Mesa 19.2.2
```

**Comment by Nobelix** [comment]
I tested it on the same device once again (GTX 1050) and the lines are now anti-aliased.

**Comment by Calinou** [comment]
This *should* be fixed in the `master` branch by https://github.com/godotengine/godot/pull/43828 – please test. This may not be fixable in 3.x due to OpenGL ES limitations.

**Comment by KoBeWi** [comment]
Bump.
@Nolkaloid Can you test if this is fixed for you in the official 4.0 alpha?
For me the lines are properly antialiased.
